tests/llms/test_huggingface.py

```diff
@@ -0,0 +1,33 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+from llama_index.llms.huggingface import HuggingFaceInferenceAPI
+
+STUB_MODEL_NAME = "placeholder_model"
+
+
+@pytest.fixture(name="hf_inference_api")
+def fixture_hf_inference_api() -> HuggingFaceInferenceAPI:
+    with patch.dict("sys.modules", huggingface_hub=MagicMock()):
+        return HuggingFaceInferenceAPI(model_name=STUB_MODEL_NAME)
```

Comment:
Self-brag: here's how one can test without using `skipif` for the module (`huggingface_hub[inferece]`) already being installed